client/src/index.jade

```diff
@@ -5,45 +5,28 @@ html
     title='TO DO'
     link(href="css/bootstrap.min.css", rel="stylesheet")
     link(href="css/style.css", rel="stylesheet")
-
-
   body
-    div(ng-app="toDoList" ng-controller="TasksCotroller")
+    div(ng-app="toDoList", ng-controller="UserController")
       div(class="navbar navbar-default")
```

Comment:
in jade it could be just `.navbar.navbar-default`. and check the rest templates please